### PR TITLE
[FEATURE] Group-based content fetching in Frontend

### PR DESCRIPTION
**Pull request by linawolf**
As  https://review.typo3.org/c/Packages/TYPO3.CMS/+/83718/2 did not make it into the Core we use the cobject viewhelper to display the content elements.

references https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/940

### CODE REVIEW

File: Resources/Private/Templates/Pages/Default.html [hunk 1]
```diff
@@ -1,24 +1,17 @@
...
-            <f:for each="{mainContent}" as="contentElement">
-                <f:cObject
-                    typoscriptObjectPath="tt_content.{contentElement.data.CType}"
-                    data="{contentElement.data}"
-                    table="tt_content"
-                />
-            </f:for>
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
```
Comment (s2b) [inline Resources/Private/Templates/Pages/Default.html:9]: This is probably obsolete for 1-column layouts

File: Resources/Private/Templates/Pages/TwoColumns.html [hunk 2]
```diff
@@ -2,27 +2,15 @@
 <f:section name="Main">
 
   <main role="main">
-    <f:render partial="Jumbotron" arguments="{jumbotronContent: jumbotronContent}"/>
+      <f:render partial="Jumbotron" arguments="{records: content.jumbotron.records}"/>
```
Comment (s2b) [inline Resources/Private/Templates/Pages/TwoColumns.html:5]: Looks like the indentation is different between template files.

File: Resources/Private/Templates/Layouts/Default.html [hunk 3]
```diff
@@ -1,3 +1,3 @@
-
 <f:render partial="Navigation/MainNavigation.html" arguments="{_all}"/>
+<f:render partial="Navigation/Breadcrumb.html" arguments="{_all}"/>
 <f:render section="Main" />
```
Comment (s2b) [inline Resources/Private/Templates/Layouts/Default.html:3]: Small style note: Self-closing tags are sometimes with and sometimes without space

Comment (linawolf) [reply Resources/Private/Templates/Layouts/Default.html]: Done

File: Resources/Private/Templates/Pages/Default.html (hunk 1 above)
Comment (linawolf) [reply Resources/Private/Templates/Pages/Default.html]: Done

**Review by s2b — APPROVED** [review]

**Pull request merged**